### ISSUE TITLE
Why are all routes appended with List ?

### ISSUE THREAD

**Issue by VincentVanclef** [acacode/swagger-typescript-api#289]
As title says, why are all the routes appended with List?

e.g.:

dashboard.intangibleValue**List**

wouldnt it make more sense to append with the request type? E.g. Post, Get?

dashboard.intangibleValue**Get**

and is there a way to change this?

**Comment by alexander-mkv** [comment]
> and is there a way to change this?

yes, you can change methodAliases inner functions for this in base route-names template
https://github.com/acacode/swagger-typescript-api/blob/master/templates/base/route-name.eta

**Comment by bekazandukeli** [comment]
> methodAliases

Url is Broken, working url:
https://github.com/acacode/swagger-typescript-api/blob/next/templates/base/route-name.ejs

**Comment by snuup** [comment]
both urls are 404. 
the question remains why in the first place replace methods with nonsensical "aliases"?

how exactly does one run the tool with adjusting the templates?
